bin/bettercap

```diff
@@ -12,34 +12,9 @@
 
 =end
 
-require 'optparse'
-require 'colorize'
-require 'packetfu'
-require 'ipaddr'
-
-Object.send :remove_const, :Config rescue nil
```

Comment:
/Library/Ruby/Gems/2.0.0/gems/bettercap-1.1.1/lib/bettercap.rb:20:in `<top (required)>': private method `remove_const' called for Object:Class (NoMethodError)
        from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require'
        from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require'
        from /Library/Ruby/Gems/2.0.0/gems/bettercap-1.1.1/bin/bettercap:15:in `<top (required)>'
        from /usr/bin/bettercap:23:in `load'
        from /usr/bin/bettercap:23:in `<main>'